<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>24</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="Control" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Control" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Control" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="VizWidget" name="openGLWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VizWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>VizWidget.hpp</header>
   <slots>
    <slot>__move(int,int)</slot>
    <slot>__rotate(int,int)</slot>
    <slot>__scale(int,int)</slot>
    <slot>showFrame(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Control</class>
   <extends>QWidget</extends>
   <header>control.h</header>
   <container>1</container>
   <slots>
    <signal>drag(int,int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>widget_3</sender>
   <signal>drag(int,int)</signal>
   <receiver>openGLWidget</receiver>
   <slot>__move(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget_2</sender>
   <signal>drag(int,int)</signal>
   <receiver>openGLWidget</receiver>
   <slot>__rotate(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>drag(int,int)</signal>
   <receiver>openGLWidget</receiver>
   <slot>__scale(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>openGLWidget</receiver>
   <slot>showFrame(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
